### PR TITLE
Add `.vscode/settings.json`

### PR DESCRIPTION
**Pull request by benfrankel**
Fixes https://github.com/TheBevyFlock/bevy-template/issues/33.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    // Allow `rust-analyzer` and `cargo` to run simultaneously.
+    // This uses a heavy amount of storage space, so consider commenting it out.
```
Comment (janhohenheim) [inline .vscode/settings.json:3]:
```suggestion
    // This uses extra storage space, so consider commenting it out.
```
It's not that heavy, as rust-analyzer doesn't do full builds.

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+    "rust-analyzer.cargo.targetDir": true
+}
```
Comment (janhohenheim) [inline .vscode/settings.json:5]: Do you have the editorconfig plugin enabled? There should be a terminating newline inserted automatically here

**Review by janhohenheim — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 2 above)
Comment (benfrankel) [reply .vscode/settings.json:5]: Nope! Installing.

File: .vscode/settings.json (hunk 1 above)
Comment (benfrankel) [reply .vscode/settings.json]: I'll take your word for it, I haven't tried this setting myself yet.

**Review by MiniaczQ — APPROVED** [review]
Works as intended

**Review by janhohenheim — APPROVED** [review]

**Pull request merged**